src/Database.php

```diff
@@ -152,6 +153,18 @@ public function __toString()
         return $this->databaseName;
     }
 
+    public function collMod($collectionOrView, $collModOption, $collModValue, array $options = [])
```

Comment:
Can we call this `modifyCollection()` as well, for consistency with `createCollection()` and `dropCollection()`?